usage_examples/el_openai/zeroshot.cfg

```diff
@@ -0,0 +1,37 @@
+[paths]
+el_nlp = null
+el_kb = null
+el_desc = null
+
+[nlp]
+lang = "en"
+pipeline = ["ner", "llm"]
```

Comment:
As we start having more complex pipelines with various components, some LLM-based and others not, I would suggest to start naming the LLM components specifically, in this case
```suggestion
pipeline = ["ner", "llm-el"]
```
Note that that would require updating all references to `llm` in this config (and the fewshot one), except for the reference to the `"llm"` factory which stays as-is.

Reply:
The reason I recommend this is because I think it will allow users to more identify compile pipelines of different parts, copy-paste parts of configs, source components, etc.